docs/network/smart-rollup-nodes.md

```diff
@@ -189,3 +189,6 @@ For example, this query gets the health of the node:
    ```
 
 Now that you have a Smart Rollup node configured for Etherlink, you can run an Etherlink EVM node, as described in [Running an Etherlink EVM node](/network/evm-nodes).
+
+When you want to stop running the Smart Rollup node, you can stop the `octez-smart-rollup-node` process and unstake the tez for the account with the `octez-client unstake` and `octez-client finalize unstake` commands.
+For more information, see [Staking mechanism](https://tezos.gitlab.io/alpha/staking.html) in the Octez documentation.
```

Comment:
Ok, thanks for the info. I've made updates here and moved this PR to draft because i need to test moving the node to bailout mode, but here's what I have so far:
https://docs-etherlink-git-unstaking-trili-tech.vercel.app/network/smart-rollup-nodes#stopping-the-smart-rollup-node

Reply:
@sribst I've updated this PR with info about bailout mode -- is this right? Preview:
https://docs-etherlink-git-unstaking-trili-tech.vercel.app/network/smart-rollup-nodes#stopping-the-smart-rollup-node